specs/proposals/blob-signing.md

```diff
@@ -0,0 +1,282 @@
+# Proposal for Supporting Blob Signing and Verification in Notation
+
+## Overview 
+
+A **blob**, short for [Binary Large Object](https://wikipedia.org/wiki/Binary_large_object) is a collection of binary data stored as a single entity. In this proposal, a blob refers to any binary data or file, including SBOM (Software Bill of Materials) files, release assets, AI model files, WebAssembly files or other forms of unstructured data. While Notation currently supports signing and verifying OCI (Open Container Initiative) artifacts such as container images, this document outlines a proposal to extend the Notation tool's capabilities to include blob signing and verification.
```

Comment:
Just some notes for AI models:
- **Ollama** --> OCI-like Artifacts
- **LM Studio** --> Hugging Face (individual files)

Reply:
Thanks.